provisioning/miniprov/hare_mp/main.py

```diff
@@ -263,7 +263,7 @@ def check_cluster_status():
         ios_cnt = 0
         for m0d in m0ds:
             if ('runs_confd' in m0d.keys() and
-                (node_info_dict[node['hostname']]['confd'][0] != 'started')):
+               (node_info_dict[node['hostname']]['confd'][0] != 'started')):
```

Comment:
Did you use auto-formatter? I would recommend using [`yapf`](https://github.com/google/yapf).

Reply:
Done. I used https://github.com/hhatto/autopep8